otx/algorithms/common/adapters/mmcv/models/builder.py

```diff
@@ -0,0 +1,21 @@
+"""Builder for OTX Common Algorithm. - mmcv.models."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from mmcv.cnn import MODELS as MMCV_MODELS
+from mmcv.utils import Registry
+
+BACKBONES = Registry("models", parent=MMCV_MODELS, scope="otx")
+TORCHVISION_BACKBONES = Registry("models", parent=MMCV_MODELS, scope="torchvision")
```

Comment:
Same with this part, maybe `Torchvision backbones` should be divided from `MMCV` family.

Reply:
If this is separated, there may be a problem with public backbone support.
This is configured to register all backbones such as `otx`, `mmcls`, `mmdet`, `mmseg`, `torchvision` backbone, etc. in the backbone registry of `mmcv` and use them at once.
Let me know if you have any other ideas :)